CalibTracker/SiStripChannelGain/src/APVGainHelpers.cc

```diff
@@ -0,0 +1,182 @@
+#include "CalibTracker/SiStripChannelGain/interface/APVGainHelpers.h"
+
+#include "DataFormats/SiStripDetId/interface/TIBDetId.h"
```

Comment:
these are deprecated functions. Use TrackerTopology. (which appears not to be a big issue from a quick investigation)